lms/static/sass/views/_shoppingcart.scss

```diff
@@ -56,6 +56,8 @@ $light-border: 1px solid $gray-l5;
 }
 
 .confirm-enrollment {
+  @include direction();
```

Comment:
You should remove this - it's the bourbon direction mixin that we don't use. Please apply the django variables like in the edx/devstack/edx-platform/lms/templates/main.html template and use the bi-app mixins from the RTL guide: https://github.com/edx/edx-platform/wiki/RTL-UI-Best-Practices

Let me know if you have questions.